control-plane/csi-driver/src/bin/node/error.rs

```diff
@@ -145,3 +148,27 @@ impl From<DeviceError> for tonic::Status {
         }
     }
 }
+
+/// Error for the fsfreeze operations.
+#[derive(Debug, Snafu)]
+#[snafu(visibility(pub(crate)), context(suffix(false)))]
+pub(crate) enum FsfreezeError {
+    #[snafu(display("Cannot find volume: volume ID: {}", volume_id))]
```

Comment:
Can do this throughout

```suggestion
    #[snafu(display("Cannot find volume: volume ID: {volume_id}"))]
```